CHANGELOG.md

```diff
@@ -5,12 +5,84 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+## [2.0.0] - 2025-07-20
+### Changed
+- **BREAKING**: Renamed core terminology throughout the codebase for better clarity, for each of the following, all functions, variables, events, and parameters have been changed.
+  - `proofSet` → `dataSet` ("proof set" becomes "data set")
+  - `root` → `piece`
+  - `rootId` → `pieceId`
+  - `owner` → `storageProvider`
+  - Function renames:
```

Comment:
this is very helpful; thanks

Reply:
I was able to use it to make a script to rename all of the functions used by curio